src/game/scenes/lobby.c

```diff
@@ -90,27 +101,30 @@ typedef struct lobby_user_t {
 } lobby_user;
 
 typedef struct lobby_local_t {
-    char name[16];
-    char helptext[80];
+    ENetHost *client;
+    ENetPeer *peer;
+    ENetPeer *opponent_peer;
     uint32_t id;
     list log;
     list users;
     bool named;
     uint8_t mode;
-    ENetHost *client;
-    ENetPeer *peer;
-    ENetPeer *opponent_peer;
     uint8_t connection_count;
     uint8_t active_user;
-    lobby_user *opponent;
     bool controllers_created;
+    lobby_user *opponent;
 
     dialog *dialog;
 
     menu *joinmenu;
 
     guiframe *frame;
     uint8_t role;
+    uint8_t nat_tries;
+    bool disconnected;
+    nat_ctx *nat;
+    char name[16];
+    char helptext[80];
```

Comment:
maybe short comments on what these variables are ? Would help reading the code. Especially "name" (name of what ?), helptext (of what ?) and nat_tries.

Reply:
Done.